src/tokens-studio/spectrum-non-colors/spectrum/typography.component/mobile.json

```diff
@@ -1239,6 +1239,46 @@
       }
     }
   },
+  "detail-cjk-size-xl": {
+    "value": "{font-size-100}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "ab476eec-b592-4890-af8f-74de808cb87f",
+        "name": "detail-size-xl"
+      }
+    }
+  },
+  "detail-cjk-size-l": {
+    "value": "{font-size-75}",
+    "type": "fontSizes",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "ab476eec-b592-4890-af8f-74de808cb87f",
```

Comment:
@larz0 not sure, where you got these uuids, but if its a complete new token in s1 (and also is a new token for s2), please remove the UUID value. Like you did in the desktop data.